docs/content/box.md

```diff
@@ -314,6 +314,19 @@ Properties:
 - Type:        string
 - Required:    false
 
+#### --box-box-config-credentials
+
+Box App config.json contents as JSON blob
+
+Leave blank normally.
+
+Properties:
+
+- Config:      box_config_credentials
+- Env Var:     RCLONE_BOX_BOX_CONFIG_CREDENTIALS
+- Type:        string
+- Required:    false
+
```

Comment:
This is in an autogenerated section, so you can remove this change from your PR.

Reply:
Done, sorry I missed the marker in the file above.

Reply:
No worries. It happens all the time :) I think we have a linter check for it now, so it would have been "flagged" when running the workflow. I just wanted to give you a chance to fix it first.